kbase-extension/static/kbase/js/util/jobLogViewer.js

```diff
@@ -527,7 +527,7 @@ define([
             scrollToEndOnNext = true;
             awaitingLog = true;
             ui.showElement('spinner');
-            bus.emit('request-latest-job-log', {
+            bus.emit('request-job-log-latest', {
```

Comment:
use `request-job-log-latest` instead of `request-latest-job-log` so that it matches all the other `request-job-*` messages